react/features/deeplinking/components/DeeplinkingDesktopPage.js

```diff
@@ -0,0 +1,158 @@
+/* @flow */
+
+import Button, { ButtonGroup } from '@atlaskit/button';
+import { AtlasKitThemeProvider } from '@atlaskit/theme';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { translate } from '../../base/i18n';
+
+import {
+    openWebApp,
+    openDesktopApp
+} from '../actions';
+
+declare var interfaceConfig: Object;
+
+/**
+ * The type of the React {@code Component} props of
+ * {@link DeeplinkingDesktopPage}.
+ */
+ type Props = {
+
+    /**
+     * Used to dispatch actions from the buttons.
+     */
+    dispatch: Dispatch<*>,
+
+    /**
+     * Used to obtain translations.
+     */
+    t: Function
+};
+
+/**
+ * React component representing unsupported browser page.
+ *
+ * @class UnsupportedDesktopBrowser
```

Comment:
You should do a pass on the comments for little typos and copy/paste errors. We both should probably switch to like webstorm which has spell checking enabled by default. I always forget to do a spellcheck pass with sublime.

Reply:
Fixed!